static/js/portico/help.js

```diff
@@ -69,6 +69,11 @@ function render_code_sections() {
     }
 }
 
+function scrollToHash(container) {
+    var hash = window.location.hash;
+    container.scrollTop = (hash !== '') ? $(hash).position().top - $('.markdown .content').position().top : 0;
```

Comment:
I'd write this with `if` statements -- when coding in an open source project, code readability is much more important than compactness, so we rarely use the `foo ? a : b` syntax.